connectors/devnull/devnull.go

```diff
@@ -116,7 +116,7 @@ func (c *Connector) CheckSchemaStatus(ctx context.Context, scope, namePrefix str
 }
 
 // CreateScope returns success
-func (c *Connector) CreateScope(ctx context.Context, scope string) error {
+func (c *Connector) CreateScope(ctx context.Context, scope, owner string) error {
```

Comment:
nit: shouldn't `owner` actually be `_` to be consistent with the schema connector?